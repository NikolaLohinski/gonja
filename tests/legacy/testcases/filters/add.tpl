{{ ["one"] | add(2) }}
{{ {"key": "value"} | add("other", "entry") }}