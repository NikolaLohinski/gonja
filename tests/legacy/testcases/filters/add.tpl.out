['one', 2]
{'key': 'value', 'other': 'entry'}